docs/development.md

```diff
@@ -81,24 +81,61 @@ glide-vc -h
 ### Using glide
 
 #### Adding new dependency
-1. Update `glide.yml` file.
 
-  Add new packages or subpackages to `glide.yml` depending if you added whole new package as dependency or
-  just new subpackage.
+1. Update `glide.yml` file
 
-2. Run `glide update --strip-vendor` to get new dependencies.
-   Than run `glide-vc --only-code --no-tests` to delete all unnecessary files from vendor.
+  Add new packages or subpackages to `glide.yml` depending if you added whole
+  new package as dependency or just new subpackage.
 
-3. Commit updated `glide.yml`, `glide.lock` and `vendor` to git.
+2. Get new dependencies
+
+```bash
+glide update --strip-vendor
+```
+
+3. Delete all unnecessary files from vendor
+
+```bash
+glide-vc --only-code --no-tests
```

Comment:
Let's keep legal files. They are really small and it would be better not to touch them.

Not sure if there is a reason for doing `--no-test-imports`